docs/reference/commandline/network_create.md

```diff
@@ -134,7 +135,13 @@ The following are those options and the equivalent docker daemon flags used for
 | `com.docker.network.bridge.enable_icc`           | `--icc`     | Enable or Disable Inter Container Connectivity        |
 | `com.docker.network.bridge.host_binding_ipv4`    | `--ip`      | Default IP when binding container ports               |
 | `com.docker.network.mtu`                         | `--mtu`     | Set the containers network MTU                        |
-| `com.docker.network.enable_ipv6`                 | `--ipv6`    | Enable IPv6 networking                                |
```

Comment:
I'm not sure what removing this from there though ? It's still a valid option when creating a bridge network right ?